core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -628,7 +628,25 @@
     - cluster: default
       namespace: ci
       name: cluster-secrets-metal
-# cluster-secrets-azure is not synced. Need to ask DPP team for this.
+#azure4
+- from:
+    pull-secret:
+      bw_item: os4-installer.openshift-ci.azure
+      attachment: pull-secret
+    osServicePrincipal.json:
+      bw_item: os4-installer.openshift-ci.azure
+      attachment: osServicePrincipal.json
+    ssh-privatekey:
+      bw_item: jenkins-ci-iam
+      attachment: ssh-privatekey
+    ssh-publickey:
+      bw_item: jenkins-ci-iam
+      attachment: ssh-publickey
+  to:
+    - cluster: default
+      namespace: ci
+      name: cluster-secrets-azure4
+# cluster-secrets-azure is not synced. Email sent to Jim about this.
 # azure:
```

Comment:
I could do the same for `cluster-secrets-azure4`.
Should I use the same BW-item `os4-installer.openshift-ci.azure` for uploading?

Reply:
We should let the Azure team self-manage `cluster-secrets-azure`

Reply:
Ack.